src/server-extension/resolvers/NftResolver/index.ts

```diff
@@ -1,15 +1,60 @@
-import { Args, Ctx, Mutation, Resolver } from 'type-graphql'
+import { Args, Ctx, Info, Mutation, Query, Resolver } from 'type-graphql'
 import { EntityManager } from 'typeorm'
-import { RequestFeaturedNftArgs, NftFeaturedRequstInfo } from './types'
+import { RequestFeaturedNftArgs, NftFeaturedRequstInfo, EndingAuctionsNftsArgs } from './types'
 import { ContextWithIP } from '../../check'
-import { withHiddenEntities } from '../../../utils/sql'
-import { NftFeaturingRequest, OwnedNft } from '../../../model'
+import { extendClause, withHiddenEntities } from '../../../utils/sql'
+import { NftFeaturingRequest } from '../../../model'
+import { OwnedNft } from '../baseTypes'
 import { randomAsHex } from '@polkadot/util-crypto'
+import { GraphQLResolveInfo } from 'graphql'
+import { Context } from '@subsquid/openreader/lib/context'
+import { model } from '../model'
+import { parseAnyTree } from '@subsquid/openreader/lib/opencrud/tree'
+import { getResolveTree } from '@subsquid/openreader/lib/util/resolve-tree'
+import { ListQuery } from '@subsquid/openreader/lib/sql/query'
 
 @Resolver()
 export class NftResolver {
   constructor(private em: () => Promise<EntityManager>) {}
 
+  @Query(() => [OwnedNft])
+  async endingAuctionsNfts(
+    @Args() args: EndingAuctionsNftsArgs,
+    @Info() info: GraphQLResolveInfo,
+    @Ctx() ctx: Context
+  ): Promise<OwnedNft[]> {
+    const tree = getResolveTree(info)
+
+    // Extract subsquid-supported Membership fields
+    const ownedNftFields = parseAnyTree(model, 'OwnedNft', info.schema, tree)
+
+    // Generate query using subsquid's ListQuery
+    const listQuery = new ListQuery(model, ctx.openreader.dialect, 'OwnedNft', ownedNftFields, {
+      limit: args.limit,
+    })
+    let listQuerySql = listQuery.sql
+
+    listQuerySql = extendClause(
+      listQuerySql,
+      'FROM',
+      `
+        INNER JOIN (
+          SELECT nft_id, auction_type, auction_type#>>'{plannedEndAtBlock}' AS end_block FROM auction a
+          WHERE auction_type#>>'{isTypeOf}' = 'AuctionTypeEnglish'
+          AND a.is_completed = false 
```

Comment:
First of all, the auction can be also canceled, in which case `is_completed` will be `false`, but `is_canceled` will be `true`, so you will need to filter out canceled auctions from the results as well. 

Another thing to note about english auctions is that they are not `completed` until settled, ie. until the winner claims the NFT, however this will always happen after the auction has already ended, ie.:
- The auction ends at `plannedEndAtBlock` and members can no longer participate
- There is an undetermined period of time before the winner actually claims the NFT
- Only after the winner claims the NFT, `is_completed` is set to `true`

So I'm not sure whether you actually want to check for `is_completed = false` or for `plannedEndAtBlock` being lower than the current block